src/test/java/tw/joi/energy/service/MeterReadingManagerTest.java

```diff
@@ -3,22 +3,22 @@
 import static java.util.Collections.emptyList;
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
-import static tw.joi.energy.fixture.ElectricityReadingFixture.createReading;
 
-import java.time.LocalDate;
+import java.time.Clock;
+import java.time.Instant;
 import java.time.ZoneId;
-import java.time.ZonedDateTime;
 import java.util.ArrayList;
 import java.util.List;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 import tw.joi.energy.domain.ElectricityReading;
 import tw.joi.energy.repository.SmartMeterRepository;
 
-public class MeterReadingManagerTest {
+class MeterReadingManagerTest {
 
     private static final ZoneId GMT = ZoneId.of("GMT");
     private static final String SMART_METER_ID = "10101010";
+    private static final Clock FIXED_CLOCK = Clock.fixed(Instant.ofEpochSecond(1721124813L), GMT);
```

Comment:
Ok, that was maybe a bit cute. I'll make that easier to understand.

What I generally wanted to avoid was the dependence on the system clock or other hidden global variables. 

You're right that it's not used in these specific tests, but it's sort of a matter of principle too 😄